common/commands.mk

```diff
@@ -32,6 +33,7 @@ VALAC_CLANG_BUILD =	valac $^ --cc=clang -D CLANG_TEST $(VALAC_FLAGS) -o $@
 VALAC_GCC_BUILD =	valac $^ --cc=gcc -D GCC_TEST $(VALAC_FLAGS) -o $@
 V_CLANG_BUILD =	v $(V_FLAGS) -cc clang -o $@ $^
 V_GCC_BUILD =		v $(V_FLAGS) -cc gcc -o $@ $^
+ZIG_BUILD = 		zig build-exe $(ZIG_FLAGS) -lc $(LIBNOTIFY_FLAGS) -femit-bin=$@ $^
```

Comment:
`libnotify` is rather a fall-back option, and it's preferred to use the native networking stack included to the language standard libraries (so the code could be as much self-contained as possible). As far as I see, it has `std.net.tcpConnectToAddress`, so I guess I'd prefer to use it.

Reply:
I'll give it a go.

Reply:
Now changed to std.net in second commit.